statistics/handle/cache/statscache.go

```diff
@@ -67,7 +67,7 @@ func (s *StatsCache) UpdateCache(newCache StatsCacheWrapper) (updated bool, newC
 	s.mu.Lock()
 	oldCache := s.cache.Load()
 	newCost = newCache.Cost()
-	if oldCache.version < newCache.version || (oldCache.version == newCache.version && oldCache.minorVersion < newCache.minorVersion) {
+	if oldCache.Version() <= newCache.Version() {
 		s.memTracker.Consume(newCost - oldCache.Cost())
 		s.cache.Store(&newCache)
 		updated = true
```

Comment:
How much performance downgrade since we do this logic more than before?

Reply:
It should be no performance regression here since whether we update the cache or not here, the `newCache` has already been created.